### ISSUE TITLE
Contour display of labels

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#1858]
## 🚀 Feature
From #1142

> ### Contour view of labels
> ... It could be useful to have a toggle that displays contours of labels instead of shaded regions. This might provided better/different clarity in some situations than just reducing opacity of the layer.

**Comment by sofroniewn** [comment]
One question here is how to make the contours if we don't depend on skimage, i'm not sure if scipy has an easy way to do this. Thoughts @jni?

**Comment by jni** [comment]
The poor man's contour is

```python
np.logical_xor(labels.astype(bool), ndi.erosion(labels.astype(bool))
```

`skimage.segmentation.find_boundaries` is just a slightly fancier version of this.

**Comment by sofroniewn** [comment]
nice, another trick will be to just do this on the part we're going to be displaying, not on the whole volume, I think

**Comment by sofroniewn** [comment]
Closed by #2168
